src/shared/ui/components/BaseTable.vue

```diff
@@ -91,6 +94,8 @@ interface Props {
   pagination: TablePagination
   items: any[]
   containerClass: string
+  breakpoint?: string | number
+  disabledPagination?: boolean
```

Comment:
Stronger types, minimal API: 

```suggestion
  pagination?: false | TablePagination
  items: any[]
  containerClass: string
  breakpoint?: string | number
```